params/default.go

```diff
@@ -14,6 +15,51 @@ func DefaultNetwork() Network {
 }
 
 func init() {
+	// check if custom network option set
+	if customNet, ok := os.LookupEnv("CELESTIA_CUSTOM_GENESIS"); ok {
+		fmt.Print("\n\nWARNING: Celestia custom network specified. Only use this option if the node is " +
+			"freshly created and initialized.\n**DO NOT** run a custom network over an already-existing node " +
+			"store!\n\n")
+
+		params := strings.Split(customNet, "=")
+		// ensure both params are present
+		if len(params) != 2 {
+			panic("must provide CELESTIA_CUSTOM_GENESIS in this format: <network_ID>=<genesis hash>")
+		}
+		netID, genHash := params[0], params[1]
+
+		// register new network and set as default for node to use
+		networksList[Network(netID)] = struct{}{}
+		genesisList[defaultNetwork] = strings.ToUpper(genHash)
+	}
+	// check if a custom network was specified
+	if network, ok := os.LookupEnv("CELESTIA_CUSTOM_NETWORK"); ok {
+		if err := Network(network).Validate(); err != nil {
+			panic("unknown network specified")
+		}
+		defaultNetwork = Network(network)
+	}
+	// check if custom bootstrappers were provided for a network
+	if bootstrappers, ok := os.LookupEnv("CELESTIA_CUSTOM_BOOTSTRAPPERS"); ok {
+		params := strings.Split(bootstrappers, "=")
+		// ensure both params are present
+		if len(params) != 2 {
+			panic("must provide CELESTIA_CUSTOM_BOOTSTRAPPERS in this format: " +
+				"<network_ID>=<boostrappers comma separated list>")
```

Comment:
Why do these custom network parameters require network ID over and over?

Reply:
If there's no need for the network ID to actually be specified each time and can instead be specified once, then that seems like better UX.

Reply:
Or we could do `CELESTIA_CUSTOM`=`networkID:genHash:bootstrapperList`

Reply:
check this: https://github.com/celestiaorg/celestia-node/pull/717/commits/c2ac8d229a329b9d6ef592cadc7e4315f55dd059

Reply:
We need them separated.

Reply:
I don't understand why the flags can't be used for that?

```
      --headers.trusted-hash string     Hex encoded header hash. Used to subjectively initialize header synchronization
      --headers.trusted-peers strings   Multiaddresses of a reliable peers to fetch headers from. (Format: multiformats.io/multiaddr)
```

Reply:
Trusted hash and trusted peers are different concepts from genesis hash and bootstrap peers. It just happens so that right now we set those from genesis and bootstrap and this will be reworked a bit with future headerex changes

Reply:
If you declare a custom network: 

`CELESTIA_CUSTOM` (must provide net name, trusted hash, and trusted peers)

if we have network already in registry, that should mean we already have bootstrappers for that network (meaning we've hard-coded it)

Lets say we have several networks available at once: 

```
testnet
testnet-2
mainnet
```

we should expose the ability to choose network via a flag and then that would take the bootstrappers for those nets from the registry (unless trusted peers flag is passed)